modules/order/templates/commerce-order--default.html.twig

```diff
@@ -0,0 +1,77 @@
+{#
+/**
+ * @file
+ * Default order template.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the wrapper.
+ * - order: The rendered order fields.
+ *   Use 'order' to print them all, or print a subset such as
+ *   'order.order_number'. Use the following code to exclude the
+ *   printing of a given field:
+ *   @code
+ *   {{ order|without('order_number') }}
+ *   @endcode
+ * - order_entity: The order entity.
+ *
+ * @ingroup themeable
+ */
+#}
+
+{{ attach_library('commerce_order/form') }}
+{% set order_state = order_entity.getState.value|capitalize %}
+
+<div class="layout-order-form clearfix">
+  <div class="layout-region layout-region-order-main">
+    {{ order.order_items }}
+    {{ order.total_price }}
+  </div>
+  <div class="layout-region layout-region-order-secondary">
+    <div class="entity-meta">
+      <div class="entity-meta__header">
+        <h3 class="entity-meta__title">
+          {{ order_state }}
+        </h3>
+        {% for key in ['completed', 'placed', 'changed'] %}
+          {% if order[key] %}
+            <div class="form-item">
+              {{ order[key] }}
+            </div>
+          {% endif %}
+        {% endfor %}
+      </div>
+      <details open>
+        <summary role="button">
+          Customer Information
+        </summary>
+        <div class="details-wrapper">
+          {% for key in ['uid', 'mail', 'ip_address'] %}
+            {% if order[key] %}
+              <div class="form-item">
+                {{ order[key] }}
+              </div>
+            {% endif %}
+          {% endfor %}
+        </div>
+      </details>
+      {% for key,value in {'billing_information': 'Billing information'|t, 'shipping_information': 'Shipping information'|t} %}
```

Comment:
It feels like it would be better for TX (themer experience) if these for loops were unrolled.
We have two elements here, repeating their HTML twice is probably no big deal?